mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -154,6 +154,20 @@ func ResourceSqlDatabaseInstance() *schema.Resource {
 							Required:    true,
 							Description: `The machine type to use. See tiers for more details and supported versions. Postgres supports only shared-core machine types, and custom machine types such as db-custom-2-13312. See the Custom Machine Type Documentation to learn about specifying custom machine types.`,
 						},
+						"advanced_machine_features": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"threads_per_core": {
+										Type:        schema.TypeInt,
+										Optional:    true,
```

Comment:
Should we make this required? We can always relax that later if more fields are added to the block. If we don't, can we test specifying `advanced_machine_features {}`?

Reply:
advanced_machine_features is optional. Added test for `advanced_machine_features {}`.